Geolocator/Geolocator/Geolocator.Plugin.Abstractions/EnergySettings.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Plugin.Geolocator.Abstractions
+{
+	/// <summary>
+	/// Energy settings for location listening (only applies to iOS).
+	/// </summary>
+	public class EnergySettings
+	{
+		/// <summary>
+		/// Gets or sets a value indicating whether the location manager should only listen for significant changes in location (approxmiately 500m), rather than continuous listening.
+		/// </summary>
+		/// <value><c>true</c> if location manager should listen for significant location updates; otherwise, <c>false</c> (default).</value>
+		public bool ListenForSignificantChanges { get; set; } = false;
+
+		/// <summary>
+		/// Gets or sets a value indicating whether the location manager should defer location updates until an energy efficient time arrives, or distance and time criteria are met. Only available on iOS 6.0 and above.
+		/// </summary>
+		/// <value><c>true</c> to defer location updates; otherwise, <c>false</c> (default).</value>
+		public bool DeferLocationUpdates { get; set; } = false;
```

Comment:
My understanding is that there are two modes of location sensing:  (1) the standard service, which is used in typical GPS-based mapping. This is a continuous monitoring mode that uses significant power. (2) The significant change service, which only reports large changes in locations and consumes less power. In the above code, `ListenForSignificantChanges` selects between these two modes.

iOS supports deferral of location updates for either of the two modes above. If deferral is not used, then any location updates (from either GPS mode above) are delivered immediately to the app, waking the app up in the case of significant changes. If deferral is used, all updates are cached until a delivery criterion (based on distance or time) is met. When this happens, all updates are delivered. In the code above, `DeferLocationUpdates` selects the deferral mode (on/off).